AFQ/api.py

```diff
@@ -924,9 +929,38 @@ def combine_profiles(self):
         out_file = op.abspath(op.join(
             self.afq_path, "tract_profiles.csv"))
         os.makedirs(op.dirname(out_file), exist_ok=True)
+        _df.reset_index(drop=True, inplace=True)
         _df.to_csv(out_file, index=False)
         return _df
 
+    def get_streamlines_json(self):
+        sls_json_fname = op.abspath(op.join(
+            self.afq_path, "tract_profiles.csv"))
+        if not op.exists(sls_json_fname):
+            first_sub = self.clean_bundles.keys()[0]
+            first_ses = self.clean_bundles[first_sub].keys()[0]
+            first_bundles_file = self.clean_bundles[first_sub][first_ses]
+            img = nib.load(self.dwi[first_sub][first_ses])
+            sft = load_tractogram(
+                first_bundles_file,
+                img,
+                Space.VOX)
+            for b in self.bundle_dict.keys():
+                if b != "whole_brain":
+                    idx = np.where(
+                        sft.data_per_streamline['bundle']
+                        == self.bundle_dict[b]['uid'])[0]
+                    # TODO: subsample idx
+                    this_tg = StatefulTractogram(
+                        sft.streamlines[idx],
+                        img,
+                        Space.VOX)
+                    # TODO: transform this_tg to MNI and rasmm
+                    # TODO: calculate core fiber
+                    # TODO: output json file
+
+        return sls_json_fname
+
     def export_all(self):
```

Comment:
this calls assemble afq browser, optionally turn off expensive things like plotly viz, afq browser, individual bundles, etc.